lntest/node.go

```diff
@@ -1599,6 +1599,15 @@ func (hn *HarnessNode) WaitForChannelPolicyUpdate(ctx context.Context,
 		select {
 		// Send a watch request every second.
 		case <-ticker.C:
+			// Did the event can close in the meantime? We want to
+			// avoid a "close of closed channel" panic since we're
+			// re-using the same event chan for multiple requests.
```

Comment:
Not really understanding this comment? would this channel get closed when `chanWatchRequests` is finished with it? Also are we always sure this is close not another channel policy update?

Reply:
I think it's that if the channel has already been closed here, and we send in another request, it'll end up double closing.

Reply:
Yes, exactly.